backend/src/main/java/com/staccato/exception/GlobalExceptionHandler.java

```diff
@@ -66,6 +66,23 @@ public ResponseEntity<ExceptionResponse> handleStaccatoException(StaccatoExcepti
                 .body(new ExceptionResponse(HttpStatus.BAD_REQUEST.toString(), e.getMessage()));
     }
 
+    @ExceptionHandler(UnauthorizedException.class)
+    @ApiResponse(responseCode = "401", content = @Content(schema = @Schema(implementation = ExceptionResponse.class)))
+    public ResponseEntity<ExceptionResponse> handleUnauthorizedErrorException(UnauthorizedException e) {
```

Comment:
위 메서드 네이밍 형식을 그대로 따라간다면, 에러 이름 앞에 handle만 붙는 식으로 "error"가 빠져야할 것 같습니다. 같이 컨벤션을 정해봐요!

```suggestion
    public ResponseEntity<ExceptionResponse> handleUnauthorizedException(UnauthorizedException e) {
```

Reply:
`InvalidServerErrorException`을 가져다 사용하다가 발생한 대참사....찾아줘서 고맙습니다:)